orttraining/orttraining/python/training/ortmodule/_io.py

```diff
@@ -554,14 +586,22 @@ def parse_outputs_for_onnx_export_and_extract_schema(
     output_names = None
     output_dynamic_axes = None
     is_deepcopy = False
+    can_deepcopied = can_module_be_deep_cloned(module, device)
```

Comment:
Can we add an env variable too to decide whether we want to deep copy or not. 
Basically here on line 589 logic should be:
do_deepcopy = if env variable DO_DEEPCOPY_DURING_EXPORT = TRUE or can_module_be_deep_cloned(module, device)

This will give us more control to decide whether we want to deepcopy.

Reply:
Default for DO_DEEPCOPY_DURING_EXPORT should be true so today's default behavior does not change

Reply:
Added.